flask/app/families.py

```diff
@@ -11,14 +11,46 @@
 @login_required
 def families_list():
 
-    db_families = models.Family.query.options(
-        joinedload(models.Family.participants)
-    ).all()
+    db_families = (
+        models.Family.query.join(models.Participant)
```

Comment:
Before the first join, add `.options(contains_eager(models.Family.participants))` and make sure to import `contains_eager` from `sqlalchemy.orm` and see if this works for the rest.